docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -2157,53 +2157,127 @@ limitation.
+as the military group quarters "household."
+
+.. todo::
+
+  Source or update these numbers.
+
+We generate (non-military) employers with an initial size attribute chosen from a skewed distribution to
+ensure that there are a few large employers and many small employers.
+Specifically, we use a log-normal distribution with :math:`\mu=4` and :math:`\sigma=1`,
+which means that the mean employer has ~90 employees and the median has ~55 employees.
+Each one has a street address, generated in the same way that residential
+addresses are generated, but without any restriction on state/PUMA/ZIP.
+
+.. todo::
+
+  This business size distribution doesn't seem plausible!
+
+.. todo::
+
+  Document how we name businesses.
+
+We generate the number of employers needed to (in expectation) employ our starting
+non-military employed population, that is:
+
+.. math::
+
+  \text{num_employers}=
+  \frac{\text{num_simulants}}{E(\text{employer_size})}=
+  \frac{\text{num_simulants}}{e^{\mu_\text{size}+\frac{1}{2}*\sigma_\text{size}}}
+
+In order to give individual simulants employers such that the size attribute is (roughly)
+accurate at the population level, we select each simulant's employer from the categorical
+distribution where the probability of each employer is proportional to its size.
+
+Finally, we set all working-age simulants living in military group quarters to be employed by the
+military.
+This is in addition to the 3% assigned across the entire working-age population regardless of
+living situation.
+
+Updating employer over time
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Working-age simulants (including those who are unemployed) change employment randomly
+at the rate of 50 changes per 100 person-years.
+This rate includes those who change from employed to unemployed.
+
+If a simulant selected for an employment change lives in military group quarters,
+no employment change occurs;
+they remain employed by the military.
+This means that in practice, the actual rate of employment changes will be a bit
+less than 50 per 100 person-years.
+
+When an employed simulant changes employment, they have a 58% chance of
+becoming unemployed and a 3% chance of becoming employed by the
+military.
+Otherwise, they sample a new non-military employer with probability
+proportional to employer size, as at initialization.
+
+In the rare case that they receive the same employer they already had,
+this whole procedure is repeated (or an equivalent mechanism
+to prevent job "changes" that don't change employment and redistribute
+the probability among all other options).
+
+The same procedure is done for an unemployed simulant selected for an employment change,
+except that instead of not being allowed to "change" to the same employer,
+they are not allowed to "change" to remain unemployed.
+
+This approach to selecting a new employer ensures that at the population level,
+the *relative* sizes of different employers will remain roughly constant (and
```

Comment:
oh oops you commented on this down here. 

i think ^ that that's not true and that the employer sizes shifted around; for example i think unemployment steadily grows(?). i ran a mini sim to test this / to show abie the weird dynamics we were going to get, but i'm sorry i don't know if i saved the results because this was originally a placeholder implementation

Reply:
Ah, okay. From reading the Slack archives, I had thought that this issue was fixed.

I can see why what we've done here will lead to a _less_ skewed distribution. If there is any easy way to check how drastic of an effect this is, let's check. But I'm not too worried about it.

However, that seems directly at odds with unemployment growing (unemployment, the single largest "employer," should shrink in a less skewed distribution), especially "steadily" which would imply no equilibrium. I can't see why what we've implemented would cause that, but if it is, I think we need to fix it.